node/src/Commands.ts

```diff
@@ -2792,6 +2792,65 @@ export function createMove(
     return createCommand(RequestType.Move, [key, dbIndex.toString()]);
 }
 
+/**
+ * @internal
+ */
+export function createDump(key: GlideString): command_request.Command {
+    return createCommand(RequestType.Dump, [key]);
+}
+
+/** Optional arguments for `RESTORE` command. */
+export type RestoreOptions = {
+    /**
+     * If set, existing key will be replaced.
+     */
+    replace?: boolean;
```

Comment:
Please Capitalize

Reply:
Why capitalize? in `createFunctionLoad`, I see `replace?` too.

Reply:
Oh sorry, I messed type and enum. We don't capitalize in types.